railties/CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
+*   Add support for stylesheets and ERB views to `rails stats`.
+
+    *Joel Hawksley*
+
 *   Allow appended root routes to take precedence over internal welcome controller.
 
     *Gannon McGibbon*
 
-
```

Comment:
This line was correct, can you put it back please?

Reply:
Will do once I figure out this rebase I just messed up ❤️